### PR TITLE
make footer responsive

### PR DESCRIPTION
**Pull request by HenryEdmodo**
i guess

### CODE REVIEW

File: public/stylesheets/style.css [hunk 1]
```diff
@@ -1,5 +1,12 @@
 body {
   font: 14px "Lucida Grande", Helvetica, Arial, sans-serif;
+/* do epic photo later
```
Comment (blakeparkinson) [inline public/stylesheets/style.css]: o god

Comment (HenryEdmodo) [inline public/stylesheets/style.css]: yeah buddy

File: public/stylesheets/style.css [hunk 2]
```diff
@@ -33,29 +40,38 @@ a {
...
+}
+ul {
+    list-style-type: none;    
 }
-.footer-list > li {
-  display: inline;
-  padding: 10px;
-  margin: 0 10px 0 10px;
+.center-it {
+	width: auto;
```
Comment (HenryEdmodo) [inline public/stylesheets/style.css]: did just this do it?

Comment (blakeparkinson) [inline public/stylesheets/style.css]: think really the only problem was i changed left float to display inline-block

**Pull request merged**